MapboxVisionFleet/VisionFleetManager/VisionFleetManager.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation
+import MapboxVision
+import MapboxVisionFleetNative
+import MapboxVisionSafety
+
+public final class VisionFleetManager {
+
+    public weak var delegate: VisionFleetManagerDelegate?
+
+    public static func create(visionManager: VisionManagerProtocol, visionSafetyManager: VisionSafetyManager) -> VisionFleetManager {
+        let manager = VisionFleetManager()
+        return manager
+    }
+
+    public func destroy() {
```

Comment:
shouldn't be called on deinit?

Reply:
It will, I'll add it when working on API.